MSVC build failure

```
 "C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Auxiliary\Build\vcvarsall.bat" x64 && "C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.40.33807/bin/Hostx64/x64/nmake.exe" /NOLOGO /S build_libs"
[build] **********************************************************************
[build] ** Visual Studio 2022 Developer Command Prompt v17.10.5
[build] ** Copyright (c) 2022 Microsoft Corporation
[build] **********************************************************************
[build] [vcvarsall.bat] Environment initialized for: 'x64'
[build] NMAKE : fatal error U1073: 不知道如何生成“"..\..\..\.cache\CPM\openssl-source\6ea3ef51de42a6e1e687c62177b17c1458a425ee\apps\apps.c"”
[build] Stop.
[build] NMAKE : fatal error U1077: “"C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\14.40.33807\bin\Hostx64\x64\nmake.exe" /LS                 depend && "C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\14.40.33807\bin\Hostx64\x64\nmake.exe" /LS                 _build_libs”: 返回代码“0x2”
[build] Stop.
[build] ninja: build stopped: subcommand failed.
```

```cmake
  if (WIN32)
      set(openssl_target_platform "VC-WIN64A")
  else()
      set(openssl_target_platform "linux-aarch64")
  endif()

  CPMAddPackage(
      NAME openssl-cmake
      GIT_TAG 3.4.0
      GITHUB_REPOSITORY jimmy-park/openssl-cmake
      OPTIONS
          "OPENSSL_TARGET_VERSION 1.1.1w" "OPENSSL_TARGET_PLATFORM ${openssl_target_platform}"
          "OPENSSL_CONFIGURE_OPTIONS no-shared;no-tests"
      )
```

--- a/cmake/ConfigureOpenSSL.cmake
+++ b/cmake/ConfigureOpenSSL.cmake
@@ -77,7 +77,7 @@ function(apply_ccache FILE)
         endif()
 
         file(READ ${FILE} MAKEFILE)
-        string(REPLACE "\nCC=" "\nCC=${CCACHE} " MAKEFILE "${MAKEFILE}")
+        string(REPLACE "\nCC=" "\nCC=ccache " MAKEFILE "${MAKEFILE}")
 
         if(MSVC)
             string(REPLACE "/Zi /Fdossl_static.pdb " "" MAKEFILE "${MAKEFILE}")

